frontend/.env.example

```
@@ -1,11 +1,35 @@
+# Backend API URL (REQUIRED)
+# Development: http://localhost:8000/api
+# Production: Your production API URL
 VITE_API_URL=http://localhost:8000/api
 
-# Optional: Enable debug mode
+# ========================================
+# OPTIONAL VARIABLES
+# ========================================
+
+# Application Name (appears in UI)
+VITE_APP_NAME=Actix Web REST API
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)